.travis.yml

```diff
@@ -127,7 +127,6 @@ before_install:
   - source ~/travis-perl-helpers/init
   - build-perl 2>/dev/null
   - perl -V
-  - npm install uglify-js@">=2.0 <3.0"
```

Comment:
Should we also add the node_modules directory to the "build cache" that Travis offers? It might save run time?

Reply:
We should even be able to remove it completely, Webpack was moved to Terser which support ES6 insteadl of uglify-js 2.x which doesn't